lib/generateMarkdown.js

```diff
@@ -24,10 +24,10 @@ function generateMarkdown(options) {
     // Verify JSON Schema version
     var schemaRef = schema.$schema;
     if (defined(schemaRef)) {
-        if (schemaRef === 'http://json-schema.org/draft-03/schema') {
```

Comment:
Correct, and other acceptable differences, like the optional '/' and '#' chars that can go on the end. The example docs use '#' there.